postgres-appliance/spilok8s.yaml

```diff
@@ -69,23 +81,30 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: spilodemo
+  name: &cluster_name spilodemo
   labels:
-    app: spilodemo
+    app: spilo
+    spilo-cluster: *cluster_name
 spec:
-  type: LoadBalancer
+  type: ClusterIP
   ports:
   - port: 5432
     targetPort: 5432
   selector:
-    app: spilodemo
+    app: spilo
+    spilo-cluster: *cluster_name
     spilo-role: master
 
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: wale-secret
+  name: &cluster_name spilodemo
+  labels:
+    app: spilo
```

Comment:
I think they are still useful. Thus we can easily find all objects related to all spilos.